### PR TITLE
#68 Interests block is not filled with purple in Profile Settings

### PR DESCRIPTION
**Pull request by karyna-kh**

### CODE REVIEW

**Review by Pavel1066 — APPROVED** [review]

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@import 'src/assets/css/styles';
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: Merge dev and use `~assets/scss/utils`

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+@include media-breakpoint-down(xl) {
+  .interests {
+    margin: 0;
+    border-radius: 0;
+  }
+
+  .interests-header {
+    background-color: #463da8;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: $gu-blue from utils

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+}
+
+.interests-content {
+  padding: 20px 28px 38px 34px;
+}
+
+@include media-breakpoint-down(xl) {
+  .interests {
+    margin: 0;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: m-0

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    border-radius: 0;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: border-0

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  .interests-header {
+    background-color: #463da8;
+
+    &__edit {
+      color: #f6f7f9;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: $gu-white

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+      border: 2px solid #f6f7f9;
+    }
+  }
+}
+
+@include media-breakpoint-down(lg) {
+  .interests-header {
+    color: #f6f7f9;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: $gu-white

also you can use text-gu-white

File: frontend/src/components/profile-settings/steps/interests-block.scss [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+    &__edit {
+      position: relative;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/steps/interests-block.scss]: position-relative

File: frontend/src/components/profile-settings/styles.scss [hunk 8]
```diff
@@ -43,6 +43,10 @@
   align-items: center;
 }
 
+.stepper__buttons {
+  justify-content: space-between;
```
Comment (nikita-remeslov) [inline frontend/src/components/profile-settings/styles.scss]: `justify-content-between ` bootstrap class

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Looks good, just need to update styles. Try to use bootstrap classes instead of custom styles

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**